Rex.xcodeproj/project.pbxproj

```diff
@@ -867,6 +871,61 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		9A7F46D01D38FE1800C42C85 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "$PROJECT_DIR/script/gyb/gyb -Dframework=UIKit -Dtarget=iOS --line-directive '' -o $SOURCE_ROOT/Source/UIKit/UIKitExtensions.iOS.generated.swift $SOURCE_ROOT/Source/Extensions.swift.gyb";
+		};
+		9A7F46D81D3900DC00C42C85 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "$PROJECT_DIR/script/gyb/gyb -Dframework=UIKit -Dtarget=tvOS --line-directive '' -o $SOURCE_ROOT/Source/UIKit/UIKitExtensions.tvOS.generated.swift $SOURCE_ROOT/Source/Extensions.swift.gyb";
+		};
+		9A7F46D91D3900E500C42C85 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "$PROJECT_DIR/script/gyb/gyb -Dframework=WatchKit -Dtarget=watchOS --line-directive '' -o $SOURCE_ROOT/Source/WatchKit/WatchKitExtensions.generated.swift $SOURCE_ROOT/Source/Extensions.swift.gyb";
+		};
+		9A7F46DA1D3900F100C42C85 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "$PROJECT_DIR/script/gyb/gyb -Dframework=AppKit -Dtarget=macOS --line-directive '' -o $SOURCE_ROOT/Source/AppKit/AppKitExtensions.generated.swift $SOURCE_ROOT/Source/Extensions.swift.gyb";
```

Comment:
Instead of using a script build phase, you should be able to create a `.gyb` build rule in the project.